src/github.com/matrix-org/dendrite/cmd/roomserver-integration-tests/main.go

```diff
@@ -336,7 +336,8 @@ func main() {
 				"state_key":"@richvdh:matrix.org",
 				"type":"m.room.member"
 			},
-			"VisibilityEventIDs":null,
+			"StateBeforeRemovesEventIDs":["$1463671339126270PnVwC:matrix.org"],
+			"StateBeforeAddsEventIDs":null,
```

Comment:
So this is effectively saying "to get the state before `$1463671339126270PnVwC:matrix.org` you need to remove `$1463671339126270PnVwC:matrix.org` and add nothing"? Which makes sense I guess.